### PR TITLE
Remove $GLOBALS['allowDeny_forbidden']

### PR DESCRIPTION
**Pull request by kamil-tekiela**
We never set this global to any truthy value. If it was meant to serve as some sort of hook then it should be implemented properly not as a global.

### CODE REVIEW

**Comment by MauricioFauth** [comment]
Looks like a bug introduced by 2327dbf3c3c109b907ea478be098c77e80e1bf0b.
After a quick look at the code, it seems like it should be true only when the parameter `$failure` is `'allow-denied'`.

File: src/Plugins/Auth/AuthenticationConfig.php [hunk 1]
```diff
@@ -95,47 +94,44 @@ public function showFailure(string $failure): never
         <tr>
             <td>';
         $config = Config::getInstance();
-        if (isset($GLOBALS['allowDeny_forbidden']) && $GLOBALS['allowDeny_forbidden']) {
```
Comment (MauricioFauth) [inline src/Plugins/Auth/AuthenticationConfig.php:98]:
```diff
- if (isset($GLOBALS['allowDeny_forbidden']) && $GLOBALS['allowDeny_forbidden']) {
+ if ($failure === 'allow-denied') {
```

Comment (MauricioFauth) [reply src/Plugins/Auth/AuthenticationConfig.php:98]: I think this fix should target 5.2.

Comment (kamil-tekiela) [reply src/Plugins/Auth/AuthenticationConfig.php:98]: Would you like to do it? You can close my PR.

**Pull request closed without merging**